src/Filter/AbstractDateFilter.php

```diff
@@ -206,6 +206,15 @@ protected function getOperator($type)
     {
         $type = (int) $type;
 
+        if (!isset(self::CHOICES[$type])) {
+            // NEXT_MAJOR: Throw an \OutOfRangeException instead.
```

Comment:
Please add the exception and comment it. Thanks

Reply:
Any idea about the right exception message ?

Reply:
Done